script/packagecloud.rb

```diff
@@ -8,9 +8,13 @@
 
 require "json"
 
+packagecloud_ruby_minimum_version = "1.0.4"
 begin
+  gem "packagecloud-ruby", ">=#{packagecloud_ruby_minimum_version}"
   require "packagecloud"
```

Comment:
I'm not a huge fan of using the exit code on L19 as control flow. How about moving stuff out of the block to look like this:

```rb
begin
  gem "packagecloud-ruby", ">=#{pkg_cloud_ruby_min_version}"
  require "packagecloud"
rescue LoadError
  puts "Requires ..."
  exit 1
end

require "packagecloud"
```